src/lib/editor.js

```diff
@@ -30,6 +30,9 @@ Editor.prototype = {
   },
 
   onSceneLoaded: function () {
+    this.sceneEl.setAttribute('debug', 'true');
```

Comment:
Yep, good point. I did this, and it's working but it doesn't reflect the changes on the DOM (It keeps showing `debug=true` but acting as `false`. It's like the `flushToDOM` doesn't do anything
```javascript
    this.sceneEl.setAttribute('debug', this.previousDebug ? 'true' : 'false');
    this.sceneEl.flushToDOM();
```